src/output/outputMod.F90

```diff
@@ -2206,7 +2206,7 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
              plocal = half*(pp1(i,j) + pp2(i,j))
 
              sensor1 = (-(fact)*(plocal-pInf))- cavitationnumber
-             sensor1 = one/(one + exp(-2*10*sensor1))
+             sensor1 = (sensor1**cavExponent)/(one + exp(-2*cavSensorSharpness*(sensor1 - cavSensorOffset)))
```

Comment:
I think it would be better to have the signs match the papers/thesis
`sensor1 = (sensor1**cavExponent)/(one + exp(2*cavSensorSharpness*(-sensor1 + cavSensorOffset)))`